lib/ansible/modules/network/fortimanager/fmgr_wanopt.py

```diff
@@ -0,0 +1,836 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: fmgr_wanopt
+version_added: "2.8"
+author:
+    - Luke Weighall (@lweighall)
+    - Andrew Welsh (@Ghilli3)
+    - Jim Huber (@p4r4n0y1ng)
+short_description: WAN optimization
+description:
+  -  Manage WanOpt security profiles in FortiManager via API
+
+options:
+  adom:
+    description:
+      - The ADOM the configuration should belong to.
+    required: false
+    default: root
+
+  host:
+    description:
+      - The FortiManager's Address.
+    required: true
+
+  username:
+    description:
+      - The username associated with the account.
+    required: true
+
+  password:
+    description:
+      - The password associated with the username account.
+    required: true
+
+  mode:
+    description:
+      - Sets one of three modes for managing the object.
+      - Allows use of soft-adds instead of overwriting existing values
+    choices: ['add', 'set', 'delete', 'update']
+    required: false
+    default: add
+
+  transparent:
+    description:
+      - Enable/disable transparent mode.
+      - choice | disable | Disable transparent mode. Client packets source addresses are changed to the source addres
+      - choice | enable | Determine if WAN Optimization changes client packet source addresses. Affects the routing
+    required: false
+    choices: ["disable", "enable"]
+
+  name:
+    description:
+      - Profile name.
+    required: false
+
+  comments:
+    description:
+      - Comment.
+    required: false
+
+  auth_group:
+    description:
+      - Optionally add an authentication group to restrict access to the WAN Optimization tunnel to peers in the authe
```

Comment:
The word - `authentication` has broken into two separate lines. Can you please fix that?